packs/pagerduty/actions/ack_incident.yaml

```diff
@@ -0,0 +1,10 @@
+name: ack_incident
+runner_type: run-python
+description: ACK an incident on PagerDuty
+enabled: true
+entry_point: ack_incident.py
+parameters:
+  event_key:
+    type: string
```

Comment:
adding type list results in error.
u'list' is not valid under any of the given schemas
Is there any other keyword for type list?